Lib/test/test_json/test_tool.py

```diff
@@ -206,3 +208,14 @@ def test_ensure_ascii_default(self):
         # asserting an ascii encoded output file
         expected = [b'{', rb'    "key": "\ud83d\udca9"', b"}"]
         self.assertEqual(lines, expected)
+
+    @unittest.skipIf(sys.platform =="win32", "The test is failed with ValueError on Windows")
```

Comment:
`The test is failed with ValueError on Windows` isn't grammatically correct.  It should be `The test failed with ValueError on Windows` (minus the `is`) 🤔

Reply:
This PR is now closed. You can write a new PR to enhance the skip message.